engine/runtime/src/test/java/org/enso/interpreter/test/ForeignMethodInvokeTest.java

```diff
@@ -0,0 +1,61 @@
+package org.enso.interpreter.test;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assume.assumeNotNull;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.file.Paths;
+import java.util.Map;
+import org.enso.polyglot.RuntimeOptions;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Engine;
+import org.graalvm.polyglot.Language;
+import org.graalvm.polyglot.Value;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ForeignMethodInvokeTest {
+  private Context ctx;
+
+  @Before
+  public void prepareCtx() {
+    Engine eng = Engine.newBuilder()
+        .allowExperimentalOptions(true)
+        .logHandler(new ByteArrayOutputStream())
+        .option(
+            RuntimeOptions.LANGUAGE_HOME_OVERRIDE,
+            Paths.get("../../distribution/component").toFile().getAbsolutePath()
+        ).build();
+    this.ctx = Context.newBuilder("enso")
```

Comment:
I tried here `this.ctx = Context.newBuilder("enso", "js")`, but it appears that even with "js" as permitted language, engine does not see "js", even though I have it installed. @hubertp Do you have any suggestions how to include "js" language here? It's not that important, I just wanted to add another test that would successfully call js foreign method, but would fail on python foreign method.